conjur/login/login_logic.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginLogic module
+
+This module is the business logic for writing user-specific configuration
+information to the user's machine
+"""
+
+# Builtins
+import logging
+
+# Internals
+import conjur.constants
+from conjur.endpoints import ConjurEndpoint
+from conjur.http import invoke_endpoint, HttpVerb
+
+class LoginLogic:
+    """
+    LoginLogic
+
+    This class holds the business logic for populating the
+    netrc configuration details needed to login to Conjur
+    """
+    credentials = None
+
+    def __init__(self, credentials):
+        self.credentials = credentials
+
+    @classmethod
+    def get_api_key(cls, ssl_verify, credential_data, password, loaded_conjurrc):
+        """
+        Method to fetch the user/host's API key from Conjur
+        """
+        params = {
+            'url': loaded_conjurrc['appliance_url'],
+            'account': loaded_conjurrc['account']
+        }
+
+        if ssl_verify is False:
+            certificate_path = False
+        # If the cert_file was provided by the user, we will use that cert to login
+        elif loaded_conjurrc['cert_file'] is None and credential_data.machine.startswith("https"):
+            certificate_path = conjur.constants.DEFAULT_CERTIFICATE_FILE
+        else:
+            certificate_path = loaded_conjurrc['cert_file']
```

Comment:
Sort this if/elif/else.
This is very unclear and hard to read.
Simplify it to explicitly demonstrate the usecases you want to handle.

Reply:
You are correct. It is not clear. 
Updating to 
```
        if ssl_verify is False:
            certificate_path = False
        elif credential_data.machine.startswith("https"):
            certificate_path = conjurrc.cert_file
```

Reply:
Thanks.
Also, `ssl_verify is False` is better read as `not ssl_verify`